docs/howto/app.rst

```diff
@@ -16,3 +16,90 @@ In the first two cases, it's important to specify the location to your tasks usi
         connector=connector,
         import_paths=["dotted.path.to", "all.the.modules", "that.define.tasks"]
     )
+
```

Comment:
The title of the page is not very relevant anymore.

Reply:
Do you mean changing 

> Find a correct place for your app

?

Reply:
yes

Reply:
I changed a few files in the doc to improve it. The open/close explanation has now its own dedicated file : `open_connection.rst`, which is the third step in the Howto : 
![image](https://user-images.githubusercontent.com/16255570/90871471-6dcfea80-e39b-11ea-9490-4dc4e38db484.png)

If this is good, we might be ready to merge :+1:

Reply:
I think it deserves its own file, as it not directly related to connector creation, nor App creation, and must be done before defining tasks.